<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformDialog1</class>
 <widget class="QDialog" name="TransformDialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>行列转置</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="dataRowBeginText">
        <property name="text">
         <string>数据集起始行数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataRowBegin"/>
      </item>
      <item>
       <widget class="QLabel" name="dataRowEndText">
        <property name="text">
         <string>数据集结束行数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataRowEnd"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="dataColBeginText">
        <property name="text">
         <string>数据集起始列数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataColBegin"/>
      </item>
      <item>
       <widget class="QLabel" name="dataColEndText">
        <property name="text">
         <string>数据集结束列数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataColEnd"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
